.gitignore

```diff
@@ -14,6 +14,7 @@ __pycache__
 .coverage
 .cache
 absolute.json
+*.swp
```

Comment:
This shouldn't have been added as part of this PR, but it's fine for now. I recommend adding vim swap files to your [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) so this doesn't come up for you in future repositories.

Reply:
Thanks, to be honest I wasn't aware of this! I can make a PR to revert this change if you want.

Reply:
I think it's ok for the moment as it won't hurt anything.